presto-main/src/main/java/io/prestosql/testing/LocalQueryRunner.java

```diff
@@ -686,7 +686,7 @@ public Lock getExclusiveLock()
             System.out.println(PlanPrinter.textLogicalPlan(plan.getRoot(), plan.getTypes(), metadata.getFunctionRegistry(), plan.getStatsAndCosts(), session, 0, false));
         }
 
-        SubPlan subplan = planFragmenter.createSubPlans(session, plan, true);
+        SubPlan subplan = planFragmenter.createSubPlans(session, plan, true, WarningCollector.NOOP);
```

Comment:
Why NOOP?

Reply:
what would you prefer to see here?

Reply:
If there's a real warning collector associated with LocalQueryRunner, that. Otherwise it's fine to keep the NOOP collector for now. I was just curious if you had looked at why it uses that.

Reply:
`LocalQueryRunner` does not have warnings collector. It already uses NOOP in other places.